src/cli.ts

```diff
@@ -55,12 +60,32 @@ export async function run(): Promise<void> {
 
   if (cliOptions.help) {
     printUsage(argsWithHelp);
+    return;
   }
-  else if (cliOptions.version) {
+  if (cliOptions.version) {
     console.log(getVersion());
+    return;
   }
-  else {
-    await startServer(options);
+
+  const serverInfos = await startServers(options);
+
+  if (serverInfos.length === 1) {
+    const urls = getServerUrls(options, serverInfos[0]!.server);
+    console.log(`Files in this directory are available under the following URLs
+    applications: ${
```

Comment:
Because that seems disruptive. If you only need one server, why start a dispatch server to bounce you to that one server?